doc/source/whatsnew/v0.23.2.txt

```diff
@@ -37,6 +37,11 @@ Documentation Changes
 -
 -
 
```

Comment:
can you add a ref here as well

Reply:
General note: IMO it is not needed to always ask this of contributors, as this ref is *only* needed when we actually want to make an explicit link to it from within the rst files (and chances are quite high we will never do this). The ref can always be added at the moment one adds a link.

Reply:
sure, but in general its a good practice